_includes/graphql/customisation.md

```diff
@@ -282,8 +282,106 @@ You can optionally override the default generated mutation names with aliases:
         "createAlias": "newRecord",
         "updateAlias": "changeRecord",
         "destroyAlias": "eraseRecord"
-      }  
+      }
     }
   ]
 }
 ```
+
+## Cloud Code Resolvers
+
+If you need more control over how your mutations modify data than what Parse's auto-generated mutations can provide, Cloud Code functions can be used as custom resolvers.
+
+For example, if you have classes named `Item` and `CartItem` in the schema, you can create an `addToCart` custom mutation that tests whether a specific item is already in the user's cart. If found, the cart item's quantity is incremented by one. If not, a new `CartItem` object is created.
+
+The ability to branch your resolver logic in this way enables you to replicate functionality found in Parse's auto-generated `createCartItem` and `updateCartItem` mutations and combine those behaviors into a single custom resolver.
+
+```sh
+# schema.graphql
+extend type Mutation {
+  addToCart(id: ID!): CartItem! @resolve
+}
+```
+
```

Comment:
Add a note/link to the getting started "Adding Custom Schema" section for better developer experience

Reply:
Will do, thanks for the reminder.